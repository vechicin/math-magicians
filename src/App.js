import React from 'react';
import Calculator from './components/Calculator';
import Navbar from './components/Navbar';
import './styles.css';

class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div className="container">
        <Navbar />
        <Calculator />
      </div>
    );
  }
}

export default App;
