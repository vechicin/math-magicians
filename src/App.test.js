import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import App from './App';

test('renders learn react link', () => {
  const linkElement = render(<BrowserRouter><App /></BrowserRouter>);
  expect(linkElement).toMatchSnapshot();
});
