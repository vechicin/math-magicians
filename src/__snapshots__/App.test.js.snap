// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <nav
          class="navBar"
        >
          <ul>
            <li>
              <a
                aria-current="page"
                class="nav-link active"
                href="/"
              >
                Home
              </a>
            </li>
            <li>
              <a
                class="nav-link"
                href="/calculator"
              >
                Calculator
              </a>
            </li>
            <li>
              <a
                class="nav-link"
                href="/quote"
              >
                Quote
              </a>
            </li>
          </ul>
        </nav>
        <div
          class="calculator-container"
        >
          <div
            class="title-container"
          >
            <h1
              class="calculator-title"
            >
              Math Magicians
            </h1>
            <h2
              class="calculator-subtitle"
            >
              Let's do some math!
            </h2>
          </div>
          <div
            class="calculator-grid"
          >
            <div
              class="output"
            >
               
               
            </div>
            <button
              type="button"
            >
              AC
            </button>
            <button
              type="button"
            >
              +/-
            </button>
            <button
              type="button"
            >
              %
            </button>
            <button
              class="operator"
              type="button"
            >
              ÷
            </button>
            <button
              type="button"
            >
              7
            </button>
            <button
              type="button"
            >
              8
            </button>
            <button
              type="button"
            >
              9
            </button>
            <button
              class="operator"
              type="button"
            >
              x
            </button>
            <button
              type="button"
            >
              4
            </button>
            <button
              type="button"
            >
              5
            </button>
            <button
              type="button"
            >
              6
            </button>
            <button
              class="operator"
              type="button"
            >
              -
            </button>
            <button
              type="button"
            >
              1
            </button>
            <button
              type="button"
            >
              2
            </button>
            <button
              type="button"
            >
              3
            </button>
            <button
              class="operator"
              type="button"
            >
              +
            </button>
            <button
              class="occupy-two"
              type="button"
            >
              0
            </button>
            <button
              type="button"
            >
              .
            </button>
            <button
              class="operator"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <nav
        class="navBar"
      >
        <ul>
          <li>
            <a
              aria-current="page"
              class="nav-link active"
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              class="nav-link"
              href="/calculator"
            >
              Calculator
            </a>
          </li>
          <li>
            <a
              class="nav-link"
              href="/quote"
            >
              Quote
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="calculator-container"
      >
        <div
          class="title-container"
        >
          <h1
            class="calculator-title"
          >
            Math Magicians
          </h1>
          <h2
            class="calculator-subtitle"
          >
            Let's do some math!
          </h2>
        </div>
        <div
          class="calculator-grid"
        >
          <div
            class="output"
          >
             
             
          </div>
          <button
            type="button"
          >
            AC
          </button>
          <button
            type="button"
          >
            +/-
          </button>
          <button
            type="button"
          >
            %
          </button>
          <button
            class="operator"
            type="button"
          >
            ÷
          </button>
          <button
            type="button"
          >
            7
          </button>
          <button
            type="button"
          >
            8
          </button>
          <button
            type="button"
          >
            9
          </button>
          <button
            class="operator"
            type="button"
          >
            x
          </button>
          <button
            type="button"
          >
            4
          </button>
          <button
            type="button"
          >
            5
          </button>
          <button
            type="button"
          >
            6
          </button>
          <button
            class="operator"
            type="button"
          >
            -
          </button>
          <button
            type="button"
          >
            1
          </button>
          <button
            type="button"
          >
            2
          </button>
          <button
            type="button"
          >
            3
          </button>
          <button
            class="operator"
            type="button"
          >
            +
          </button>
          <button
            class="occupy-two"
            type="button"
          >
            0
          </button>
          <button
            type="button"
          >
            .
          </button>
          <button
            class="operator"
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
